compute/elastic-metal/how-to/index.mdx

```diff
@@ -1,8 +1,141 @@
 ---
 meta:
-  title: Elastic Metal servers - How Tos
-  description: Elastic Metal How Tos
+  title: Setting up Gitlab with a Managed Database for PostgreSQL
```

Comment:
Seems like something has gone wrong here? I think this is the wrong content for this file.